web/router.ex

```diff
@@ -7,7 +7,6 @@ defmodule Booky.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    plug :assign_current_user
```

Comment:
I think this might be necessary for other controllers/views to get access to the logged in user.